### DISCUSSION TITLE
绘图完成后的文件硬盘能不能拔掉，等需要耕种时再插上

### DISCUSSION THREAD

**Discussion by zyf456123789** [Chia-Network/chia-blockchain#2157] [Q&A] [answered]
我的绘图文件是保存在移动硬盘中，但是有时要调整USB接口，正在耕种的文件能不能拔掉硬盘，等需要时再插上继续耕种。还是说一旦耕种，就不能拔掉硬盘。

**Comment by cccat6** [comment, marked as answer]
可以拔掉，Chia挖矿的时候访问硬盘的频率很低。只要不是正好撞在挖矿的时候就没啥影响。就算plot掉了，重新刷新加载一下就行了。
